doc/api/cli.md

```diff
@@ -230,6 +230,14 @@ https://github.com/tc39/ecma262/pull/1320.
 
 Both of the above may change in future updates, which will be breaking changes.
 
+### `--heapsnapshot-signal=signal`
```

Comment:
Please also add to the list of options allowed in `NODE_OPTIONS` later on in this file.